lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -324,6 +379,44 @@ bool AsyncPipelinedClient::add_gy_final_action_flow(
   return true;
 }
 
+ void AsyncPipelinedClient::set_upf_session_rpc(
+     const SessionSet& request,
+     std::function<void(Status, UpfRes)> callback) {
+   auto local_resp = new AsyncLocalResponse<UpfRes>(
+       std::move(callback), RESPONSE_TIMEOUT);
+   PrintGrpcMessage(
+       static_cast<const google::protobuf::Message&>(request));
+   std::cerr << __LINE__ << " "<< __FUNCTION__ << " GRPC message sending to Upf client details\n";
```

Comment:
Why are we logging this way? Is this only for testing?

Reply:
Yes Marie, this is because we are using unit test scripts to invoke the session creation and the relevant details are not published in syslog. We'll remove it as invoking via python scripts solves this issue. 

Thanks